tests/test_builders/test_build_epub.py

```diff
@@ -197,22 +212,24 @@ def test_epub_cover(app):
 
 
 @pytest.mark.sphinx('epub', testroot='toctree')
-def test_nested_toc(app):
+def test_nested_toc(app: SphinxTestApp) -> None:
     app.build()
 
     # toc.ncx
     toc = EPUBElementTree.fromstring((app.outdir / 'toc.ncx').read_bytes())
     assert toc.find('./ncx:docTitle/ncx:text').text == 'Project name not set'
 
     # toc.ncx / navPoint
-    def navinfo(elem: EPUBElementTree):
```

Comment:
`navinfo` was defined twice in this test, which `mypy` does not like. I renamed both definitions.